aiven/client/cliarg.py

```diff
@@ -2,15 +2,16 @@
 #
 # This file is under the Apache License, Version 2.0.
 # See the file `LICENSE` for details.
-
-from .argx import arg, UserError
+# type: ignore[attr-defined]
+from .argx import arg, CommandLineTool, UserError
 from functools import wraps
+from typing import Any, Callable
 
 import json as jsonlib
 import os
 
 
-def get_json_config(path_or_string):
+def get_json_config(path_or_string: str) -> Any:
```

Comment:
Shouldn't the result be restricted to `Dict` (or maybe even `Dict[str, Any]`) instead?